cmd/backup.go

```diff
@@ -170,6 +176,12 @@ func runBackup(flagSet *pflag.FlagSet, cmdName, db, table string) error {
 	if err != nil {
 		return err
 	}
+
+	content, err := client.GetBackupTS(ctx)
+	if err != nil {
+		return err
+	}
+	fmt.Println(string(content))
```

Comment:
I don't think we still need to log the backup TS if we write it to a human-readable file.